### PR TITLE
week-0-b

### PR DESCRIPTION
**Pull request by karenl7**
Another example to test work flow.

### CODE REVIEW

File: bite-size-notes/week-0-b-test.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## new notes
+
+Cats are cute.
```
Comment (ctrl-admin) [inline bite-size-notes/week-0-b-test.md]: This should be `Cats are the cutest.`

File: bite-size-notes/week-0-b-test.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Boba is yummy.
+
+Another photo of a cat:
```
Comment (ctrl-admin) [inline bite-size-notes/week-0-b-test.md]: But this is the same picture as before. Can you find a different photo to make the argument more convincing.

**Review by ctrl-admin — CHANGES_REQUESTED** [review]
Some feedback
- what you liked
- general areas of improvement

File: bite-size-notes/week-0-b-test.md (hunk 1 above)
Comment (karenl7) [reply bite-size-notes/week-0-b-test.md]: I have made the necessary changes to address this comment.

File: bite-size-notes/week-0-b-test.md (hunk 2 above)
Comment (karenl7) [reply bite-size-notes/week-0-b-test.md]: I agree. I found another cute photo.

**Pull request merged**